packaging/build_windows.sh

```diff
@@ -6,13 +6,32 @@ sudo apt-get update
 sudo apt-get install -y --no-install-recommends \
      g++-mingw-w64-i686 mingw-w64-i686-dev g++-mingw-w64-x86-64 \
      mingw-w64-x86-64-dev build-essential libtool autotools-dev automake pkg-config \
-     libssl-dev libevent-dev bsdmainutils curl ca-certificates
+     libssl-dev libevent-dev bsdmainutils curl ca-certificates libicu-dev
+
+#################################################################
+# Build ICU for Linux first so that we can cross compile it below
+#################################################################
+staging_dir=/tmp/icu_staging
+icu_linux_dir=$staging_dir/build_icu_linux
+mkdir -p $staging_dir
+pushd $staging_dir
+wget http://download.icu-project.org/files/icu4c/55.1/icu4c-55_1-src.tgz
```

Comment:
boost.locale does not require this? Why would we need to get this separately from boost?

Reply:
Because we're using icu for normalization, which doesn't work through boost unless it's explicitly enabled.

Reply:
So what is the gain with using boost's wrapper? Suppose we just make ICU calls directly?

Reply:
It's C++ and style-compatible.  I also have previous experience using it, for exactly this kind of purpose.